CONTRIBUTING.md

```diff
@@ -150,6 +167,22 @@ To test a specific feature, specify the full path to the function to run:
 pytest -vs tests/integration/synapseclient/test_command_line_client.py::test_table_query
 ````
 
+#### Integration testing against the `dev` synapse server
+The easiest way to specify the HTTP endpoints to use for all synapse requests is to modify the `~/.synapseConfig` file and modify a few key-value pairs such as below. Not this is also where you will specify the dev authentication:
+```
+[authentication]
+username=<dev username>
+authtoken=<dev authtoken>
+
+[endpoints]
+repoEndpoint=https://repo-dev.dev.sagebase.org/repo/v1
+authEndpoint=https://repo-dev.dev.sagebase.org/auth/v1
+fileHandleEndpoint=https://repo-dev.dev.sagebase.org/file/v1
+```
+
+#### Integration testing for external collaborators
+As an external collaborator you will not have access to a development account and environment to run the integration tests agaisnt. Either request that a Sage Bionetworks staff member run your integration tests via a pull request, or, contact us via the [Help Forum](https://www.synapse.org/#!SynapseForum:default) to requisition a development account for integration testing only.
+
```

Comment:
Instead of directing users to the forum, please direct them to an Atlassian help desk, either the Synapse help desk https://sagebionetworks.jira.com/servicedesk/customer/portal/9 or a different one owned by D&T.

Reply:
Added this change via: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/997/files